tests/ttnn/unit_tests/operations/test_conv2d.py

```diff
@@ -747,6 +752,9 @@ def test_sd_conv(
         )
 
 
+@skip_for_wormhole_b0(
```

Comment:
I do not expect SD conv unit tests to fail on WH. The SD model runs on WH and there are no l1 fitting issues there.

Reply:
We synced offline.. @tt-nshanker mentioned some of these failing permutations are easy fixes and will follow-up on some of these so we can add them to the test-coverage